pkg/vault/contracts/CompositeLiquidityRouterHooks.sol

```diff
@@ -310,7 +310,7 @@ abstract contract CompositeLiquidityRouterHooks is BatchRouterCommon {
         }
 
         if (actualAmountIn > maxAmountIn) {
-            revert IVaultErrors.AmountInAboveMax(settlementToken, amountIn, maxAmountIn);
+            revert IVaultErrors.AmountInAboveMax(settlementToken, actualAmountIn, maxAmountIn);
```

Comment:
right; I think we didn't catch it because it's actually impossible to get here in the wrong case (`erc4626BufferWrapOrUnwrap` should revert first).